### PR TITLE
Meeting stage view C#-> React -> Live share SDK

### PR DESCRIPTION
**Pull request by HunaidHanfee-MSFT**

### CODE REVIEW

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/components/app-in-meeting.jsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    const shareSpecificPart = (partName) => {
+        var appContentUrl = "";
+        microsoftTeams.app.getContext().then((context) => {
+          appContentUrl = partName == 'todo' ? `${window.location.origin}/todoView?meetingId=${context.meeting.id}` : partName == 'doing' ? `${window.location.origin}/doingView?meetingId=${context.meeting.id}` : `${window.location.origin}/doneView?meetingId=${context.meeting.id}`;
+          microsoftTeams.meeting.shareAppContentToStage((err, result) => {
+            if (result) {
+              // handle success
+              console.log(result);
+            }
+            if (err) {
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/components/app-in-meeting.jsx:46]: One line space before starting if

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/components/configure.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            microsoftTeams.app.notifySuccess();
+            microsoftTeams.pages.config.registerOnSaveHandler(function (saveEvent) {
+                var radios = document.getElementsByName('notificationType');
+                microsoftTeams.pages.config.setConfig({
+                    entityID: "App in Meeting Demo",
+                    contentUrl: `${window.location.origin}/appInMeeting`,
+                    suggestedTabName: "App in meeting",
+                    websiteUrl: `${window.location.origin}/appInMeeting`,
+                });
+                saveEvent.notifySuccess();
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/components/configure.jsx:22]: Add proper line break and spacing.

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/components/configure.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    const onClick = () => {
+        microsoftTeams.pages.config.setValidityState(true);
+    }
+
+    return (
+        <header className="header">
+            <div className="header-inner-container">
+                <div id="divConfig">
+                    <br />
+                    <input type="radio" name="notificationType" value="Create" onClick={onClick}/> Add hun in a meeting
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/components/configure.jsx:36]: hun ?

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/components/doing.jsx [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+
+        let details = [...meetingDataArray];
+        details.push(meeting);
+
+        saveMeetingData(meeting, meetingId);
+
+        editorMap.set(editorValueKey, details);
+    }
+
+
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/components/doing.jsx:111]: Remove empty extra line

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/components/doing.jsx [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+        if (meetingDataArray) {
+            meetingDataArray.forEach((meeting) => {
+                elements.push(<div className="details">
+                    <div className="description" title={meeting.taskDescription}>{meeting.taskDescription}</div>
+                    <div className="userName">{meeting.userName}</div>
+                </div>);
+            })
+        }
+        return elements;
+    }
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/components/doing.jsx:145]: Add proper line break and spacing.

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/components/done.jsx [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+        let elements = [];
+        if (meetingDataArray) {
+            meetingDataArray.forEach((meeting) => {
+                elements.push(<div className="details">
+                    <div className="description" title={meeting.taskDescription}>{meeting.taskDescription}</div>
+                    <div className="userName">{meeting.userName}</div>
+                </div>);
+            })
+        }
+        return elements;
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/components/done.jsx:144]: Add proper line break and spacing

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my comments

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/components/done.jsx (hunk 6 above)
Comment (HunaidHanfee-MSFT) [reply samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/components/done.jsx:144]: Done

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/components/doing.jsx (hunk 5 above)
Comment (HunaidHanfee-MSFT) [reply samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/components/doing.jsx:145]: Done

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**